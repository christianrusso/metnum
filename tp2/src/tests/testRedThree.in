../data/ImagenesCarasRed/ 28 23 4 5 10
s1/ 1 2 3 4 5
s2/ 1 2 3 4 5
s3/ 1 2 3 4 5
s4/ 1 2 3 4 5
7
../data/ImagenesCarasRed/s1/10.pgm 1
../data/ImagenesCarasRed/s2/10.pgm 2
../data/ImagenesCarasRed/s3/10.pgm 3
../data/ImagenesCarasRed/s4/10.pgm 4
../data/ImagenesCarasRed/s5/10.pgm 5
../data/ImagenesCarasRed/s1/1.pgm 1
../data/ImagenesCarasRed/s2/1.pgm 2