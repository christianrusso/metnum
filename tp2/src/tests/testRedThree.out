530.9959534
336.2080296
267.9688944
256.044916
164.3221685
159.6156182
151.1633476
129.1976019
126.4633382
123.667324
